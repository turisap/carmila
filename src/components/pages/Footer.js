/**
 * Created by HP on 09-Dec-17.
 */
import React from 'react';
import {NavLink} from 'react-router-dom';

const Footer = () => (
    <div className="footer__fix">

        <footer>
            <ul className="footer__links">
                <li><NavLink className="footer__link text-center" to="/about">About Us</NavLink></li>
                <li><NavLink className="footer__link text-center" to="#">Sign Up For News</NavLink></li>
            </ul>
            <ul className="footer__info">
                <li>
                    <p id="footer__info">Our office is located 24, Rd No 2 Krong Battambang.
                        We are open 8a.m. to 5p.m Monday to Friday.</p>
                </li>
            </ul>
        </footer>
    </div>
);

export default Footer;