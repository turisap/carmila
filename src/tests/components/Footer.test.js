/**
 * Created by HP on 15-Dec-17.
 */
import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/pages/Footer';

test('should render footer properly', () => {
   const wrapper = shallow(<Footer/>);
   expect(wrapper).toMatchSnapshot();
});