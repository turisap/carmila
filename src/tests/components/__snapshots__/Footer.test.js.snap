// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer properly 1`] = `
<div
  className="footer__fix"
>
  <footer>
    <ul
      className="footer__links"
    >
      <li>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="footer__link text-center"
          to="/about"
        >
          About Us
        </NavLink>
      </li>
      <li>
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="footer__link text-center"
          to="#"
        >
          Sign Up For News
        </NavLink>
      </li>
    </ul>
    <ul
      className="footer__info"
    >
      <li>
        <p
          id="footer__info"
        >
          Our office is located 24, Rd No 2 Krong Battambang. We are open 8a.m. to 5p.m Monday to Friday.
        </p>
      </li>
    </ul>
  </footer>
</div>
`;
